src/modules/iotjs_module_i2c.h

```diff
@@ -44,14 +46,14 @@ typedef enum {
 typedef struct {
 #if defined(__linux__) || defined(__APPLE__)
   iotjs_string_t device;
-#elif defined(__NUTTX__)
+#elif defined(__NUTTX__) || defined(__TIZENRT__)
   int device;
 #endif
   char* buf_data;
+  int32_t delay;
```

Comment:
Wo don't 'really' need.
It is just a good practice to keep similarly sized struct members next to one another.
This generally helps performance: seed, code size and memory footprint.